frontend/public/components/start-guide.tsx

```diff
@@ -1,130 +1,134 @@
+import { RootState } from '../redux';
+import { useFlag } from '@console/shared/src';
+import { ExternalLinkAltIcon, OpenshiftIcon } from '@patternfly/react-icons';
 
-export const OpenShiftGettingStarted = connect(createProjectMessageStateToProps)(
-  ({ canCreate = true, createProjectMessage }: OpenShiftGettingStartedProps) => {
-    const { t } = useTranslation();
-    const [, setActiveNamespace] = useActiveNamespace();
-    const [perspective] = useActivePerspective();
-    const createNamespaceOrProjectModal = useCreateNamespaceOrProjectModal();
+export const OpenShiftGettingStarted: React.FCC<OpenShiftGettingStartedProps> = ({ title }) => {
+  const { t } = useTranslation();
+  const [, setActiveNamespace] = useActiveNamespace();
+  const [perspective] = useActivePerspective();
+  const canCreateNamespace = useFlag(FLAGS.CAN_CREATE_NS);
+  const canCreateProject = useFlag(FLAGS.CAN_CREATE_PROJECT);
+  const canCreate = canCreateNamespace || canCreateProject;
+  const createProjectMessage = useSelector(({ UI }: RootState) => UI.get('createProjectMessage'));
+  const createNamespaceOrProjectModal = useCreateNamespaceOrProjectModal();
+  const onClickCreate = () =>
+    createNamespaceOrProjectModal({
+      onSubmit:
+        perspective !== 'admin'
+          ? (project: K8sResourceKind) => {
+              setActiveNamespace(project.metadata?.name);
+            }
+          : undefined,
+    });
+
+  const primaryActions = canCreate
+    ? [
+        <Button key="create-project-action" variant={ButtonVariant.primary} onClick={onClickCreate}>
+          {t('public~Create a new Project')}
+        </Button>,
+      ]
+    : [];
 
-    return (
-      <>
-        {canCreate ? (
-          <p>
-            {t(
-              'public~OpenShift helps you quickly develop, host, and scale applications. To get started, create a project for your application.',
-            )}
-          </p>
-        ) : (
-          <p>
-            {t(
-              "public~OpenShift helps you quickly develop, host, and scale applications. To get started, you'll need a project. Currently, you can't create or access any projects.",
-            )}
-            {!createProjectMessage && (
-              <>&nbsp;{t("public~You'll need to contact a cluster administrator for help.")}</>
-            )}
-          </p>
-        )}
-        {createProjectMessage && (
-          <p className="co-pre-line">
-            <LinkifyExternal>{createProjectMessage}</LinkifyExternal>
-          </p>
-        )}
+  const secondaryActions = [
+    <Button
+      key="download-cli-tools"
+      variant={ButtonVariant.link}
+      component="a"
+      href="/command-line-tools"
+    >
+      {t('public~Download command-line tools')}
+    </Button>,
+    <Button
+      key="visit-docs"
+      variant={ButtonVariant.link}
+      component="a"
+      href={openshiftHelpBase}
+      target="_blank"
+      rel="noopener noreferrer"
+      icon={<ExternalLinkAltIcon />}
```

Comment:
to follow existing `co-external-link` design

```suggestion
      icon={<ExternalLinkAltIcon />}
      iconPosition="end"
```